src/platform/platform_activation.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "pal.h"
+#include <string_view>
+
+namespace xlang::impl
+{
+#if XLANG_PLATFORM_WINDOWS
```

Comment:
should this be in pal.h?

Reply:
PAL.h has xlang_get_activation_factory. Pretty sure this file is explicitly part of the implementation.